aspnet/mvc/models/validation/sample/UsersController.cs

```diff
@@ -0,0 +1,22 @@
+using Microsoft.AspNet.Mvc;
+using Microsoft.AspNet.Mvc.Rendering;
+using System.Text.RegularExpressions;
+
+namespace MVCMovie.Controllers
+{
+    [Route("[controller]/[action]")]
```

Comment:
I would setup the default MVC route in startup instead of using attribute routing here.